Use cases/UseCases.md

```diff
@@ -135,6 +138,22 @@ Results in a response with a cause of death registgration summary like this:
 }
 ```
 
+The cause of death registration is currently available as a legal document "Legeerklæring" PDF in Elektronisk dødsmelding Web at https://{environment}.helsepunkt.no/dodsmelding/mineregistreringer. This document is only available for the doctor who completed the registration.
+
+## Change time of death on existing death message
+A change in time of death on a Patient can be done by any authorized doctor. 
+It can be done following the same steps as for register death message only the Patient will during search Patient by id or by identifier be registered as dead. During create of Patient enter the new time of death. 
```

Comment:
Forslag: To overwrite the existing time of death, simply register a new death message on the same patient.